src/extension.ts

```diff
@@ -48,7 +48,10 @@ export async function activate(context: vscode.ExtensionContext) {
         context.subscriptions.push(uiExtensionVariables.outputChannel);
 
         registerAzureUtilsExtensionVariables(uiExtensionVariables);
-
+        vscode.commands.executeCommand(
+            'workbench.action.openWalkthrough', 
+            { category: 'ms-kubernetes-tools.vscode-aks-tools#walkthrough'}
```

Comment:
Tats, everything looks good. Instead of keeping this as generic name "walk through" do we want to name it as "aks tools : walk through" or "aks tools: welcome page" something tied to "aks" ? Please let me know if this make sense. Thanks.

Reply:
Thank you for this, I see what you mean I could simplify to make it clear essentially the way this thing gets mapped it’s the combination of “publishername.extensionid#id-of-thewalkthrough-tag” but instead of string I could read this through meta tag and make that more clear. So, essentially in this the initial objects are not changeable in name but I will play around and I’ll see if I can simplify those sentiments using meta information ℹ️☕️🥷

Reply:
As discussed, let's keep it as ```walkthrough``` as id, my intention was to make this searchable but seems like it wouldn't help. Thanks for the discussion.